spec/test.js

```diff
@@ -0,0 +1,52 @@
+"When we run getHeadlines, it puts headline contents in page"
+//----------------------------------------------------
+//arrange
+//----------------------------------------------------
+const stubbedTitle = "random picture";
+const stubbedWebUrl = "https://www.google.com";
+const stubbedApiUrl = "https://www.facebook.com/";
+const stubbedHeadlines = { response: { results: [ { webTitle: stubbedTitle, webUrl: stubbedWebUrl, apiUrl: stubbedApiUrl } ] } };
+const stubbedSummary = { text: "summary text", sentences: "summary sentences" };
+const stubbedImgUrl = "https://picsum.photos/200";
+const stubbedArticle = { response: { content: { fields: { body: `<div><img src="${stubbedImgUrl}"></div>`} } } }
+
```

Comment:
This is nice, Iike how you've taken the elements of the fetch response you needed and stubbed them in a test file